.github/workflows/test.yml

```diff
@@ -79,29 +79,18 @@ jobs:
           name: main.breakdown
           path: main.breakdown
           if-no-files-found: error
-
-      - name: find pull request ID
-        run: |
-          PR_DATA=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            "https://api.github.com/repos/${{ github.repository }}/pulls?head=${{ github.repository_owner }}:${{ github.ref_name }}&state=open")
-          PR_ID=$(echo "$PR_DATA" | jq -r '.[0].number')
```

Comment:
@vladopajic this line returns nothing, because `head=${{ github.repository_owner }}` is incorrect

```
curl -s -H 'Authorization: token ****' 'https://api.github.com/repos/vladopajic/go-test-coverage/pulls?head=vladopajic:184/merge&state=open'
[

]
```

Reply:
based on docu https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests it must be:
```
curl -s -H 'Authorization: token ****' 'https://api.github.com/repos/vladopajic/go-test-coverage/pulls?state=open' | jq -r '.[0].number'
184
```

Reply:
Yeah it behaves differently. I think we have different GH on statement.